Sources/TestWebService/main.swift

```diff
@@ -42,15 +42,22 @@ struct TestWebService: Apodini.WebService {
     }
     
     struct Greeter: Component {
```

Comment:
I would suggest to add a test case that showcases how `Injectables`  or `Dynamics` can be dynamically created in the initializer as shown on slide 7 in @hendesi 's slides: https://confluence.ase.in.tum.de/display/SERVERSIDESWIFT2021/Intermediate+Presentation?preview=/66506934/66513049/Intermediate_Presentation_Felix_Desiderato.pdf

Reply:
I cannot do it exactly as suggested by @hendesi. I think the `Dynamics`' `wrappedValue` should be get-only, as its properties are to be fixed at startup. The closest I can get is the following (with the only difference being the underscore in `self._dynamics`):

```swift
struct HandlingComponent: Component {
    @Dynamics var dynamics: [String: Param<String>]
        
    init() {
        self._dynamics = ["name": Param<String>()]
    }
        
    func handle() -> String {
        dynamics["name"]
    }
}
```